packages/@aws-cdk/aws-lambda/test/code.test.ts

```diff
@@ -245,6 +245,29 @@ describe('code', () => {
       });
     });
 
+    test('digests are interpreted correctly', () => {
```

Comment:
I'd like to see a test that confirms that tags are not misinterpreted as digests as well.

Reply:
The existing `props are correctly resolved` test (previous in the file) covers that case I think; I can add an explicitly covering the tag case if you prefer.

Reply:
Ah I see, you're right. That's fine then, no need to add an extra test.